src/gennylib/include/gennylib/context.hpp

```diff
@@ -60,6 +60,11 @@ class HasNode {
 public:
     explicit HasNode(const Node& node) : _node{node} {}
 
+    template <typename... Args>
```

Comment:
Just noting that this is just a magic incantation to me. Someone else will need to sanity check this part.

Reply:
lol ya np - it's copy/paste from the lines below - saved me from having to write 

```c++
context.operator[]("ClientName")
```

😹